src/test/java/api/requests/scenarios/MoveRequestTests.java

```diff
@@ -205,6 +204,87 @@ void itemShouldRemainPagedIfHoldCreatedAfterRequestHasBeenMovedToAnotherItem() {
     assertThat(itemsClient.get(firstItem), hasItemStatus(PAGED));
   }
 
+  @Test
+  void canMovePageTlrToAvailableItem() {
+    configurationsFixture.enableTlrFeature();
+
+    val firstItem = itemsFixture.basedUponSmallAngryPlanet("89809");
+    val pageIlrForFirstItem = requestsFixture.placeTitleLevelPageRequest(firstItem.getInstanceId(),
+      usersFixture.james());
+    ItemResource secondItem = itemsFixture.basedUponSmallAngryPlanet(
+      holdingBuilder -> holdingBuilder,
+      instanceBuilder -> instanceBuilder.withId(firstItem.getInstanceId()),
+      itemBuilder -> itemBuilder
+    );
+
+    requestsFixture.move(new MoveRequestBuilder(pageIlrForFirstItem.getId(),
+      secondItem.getId()));
+    assertThat(itemsClient.get(firstItem), hasItemStatus(AVAILABLE));
+    assertThat(itemsClient.get(secondItem), hasItemStatus(PAGED));
+  }
+
+  @Test
+  void canMovePageTlrToRecall() {
+    configurationsFixture.enableTlrFeature();
+    val firstItem = itemsFixture.basedUponSmallAngryPlanet("89809");
```

Comment:
a blank line(or remove them from the other tests) for consistency of code style?